.github/workflows/deploy_docs.yml

```diff
@@ -34,14 +34,14 @@ jobs:
         run: |
               wget https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh
               bash elan-init.sh -y
-      - name: Download doc-gen4
+      - name: Download dependencies
         working-directory: ./cedar-lean
         shell: bash
-        run: source ~/.profile && sed -i '/^meta if get_config? env = some "dev" then/d' lakefile.lean && lake update doc-gen4
+        run: source ~/.profile && lake -R -Kenv=dev update
       - name: Build docs
         working-directory: ./cedar-lean
         shell: bash
-        run: source ~/.profile && lake -Kenv=dev build Cedar:docs
+        run: source ~/.profile && lake -R -Kenv=dev build Cedar:docs
```

Comment:
for my curiosity, why is `-R` required?

Reply:
Tbh I'm not sure. It wasn't working without the `-R` in `ci.yml` so I figured I might as well make the change in `deploy_docs.yml` too. All the instructions on https://github.com/leanprover/doc-gen4 use that flag, which is why I tried it.

Here's the command description: `--reconfigure, -R     elaborate configuration files instead of using OLeans`